<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1036</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>351</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>Добавить книгу в каталог</string>
    </property>
    <widget class="QLineEdit" name="titleLineEdit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>161</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="authorLineEdit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>161</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="yearLineEdit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>130</y>
       <width>161</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="addBookPushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>301</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Название книги           --&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Автор книги                 --&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Год выпуска книги     --&gt;</string>
     </property>
    </widget>
    <zorder>titleLineEdit</zorder>
    <zorder>authorLineEdit</zorder>
    <zorder>yearLineEdit</zorder>
    <zorder>label</zorder>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>addBookPushButton</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>651</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>Результаты</string>
    </property>
    <widget class="QPushButton" name="deletePushButton">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>130</y>
       <width>181</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить книгу из каталога</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>441</width>
       <height>351</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="sortingPushButton">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>70</y>
       <width>181</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Сортировать по автору</string>
     </property>
    </widget>
    <widget class="QPushButton" name="findPushButton">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>100</y>
       <width>181</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Поиск книги в каталоге</string>
     </property>
    </widget>
    <widget class="QPushButton" name="uploadPushButton">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>40</y>
       <width>181</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Обновить</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="resetPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>1011</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1036</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
